hub3/ead/nodes.go

```diff
@@ -190,6 +193,41 @@ func CreateTree(cfg *NodeConfig, n *Node, hubID string, id string) *fragments.Tr
 	tree.HasRestriction = n.AccessRestrict != ""
 	tree.PhysDesc = n.Header.Physdesc
 
+	if tree.HasDigitalObject {
+		daoCfg := newDaoConfig(cfg, tree)
+
+		// must happen here because the check needs the daoCfg to not be written yet
+		hasOrphanedMetsFile := daoCfg.hasOrphanedMetsFile()
+
+		if err := daoCfg.Write(); err != nil {
+			log.Error().Err(err).Msg("unable to write daocfg to disk")
```

Comment:
Is it ok to continue after this error?

Reply:
Here it is ok. When we can't write it we can still continue to process the record. And we will try to store it during the next pass.